server/requestHandler.js

```diff
@@ -98,6 +101,11 @@ exports.userAddNotes = (req, res) => {
       if(err) {
         res.status(404).send('Could not find user.');
       }
+
+      if(!user) {
+        return addNewUser(req, res);
```

Comment:
Anyway we can make it so that the user is added AND the note is saved as well? This is so that the user does not have to highlight the text a second time to actually save it.